src/main/scala/org/deeplearning4j/scalphagozero/board/GameState.scala

```diff
@@ -87,51 +89,13 @@ class GameState(
       }
     }
 
-  def isOver: Boolean =
-    this.lastMove match {
-      case None | Some(Move.Play(_)) => false
-      case Some(Move.Resign)         => true
-      case Some(Move.Pass) =>
-        val secondLastMove = this.previousState.get.lastMove
-        secondLastMove match {
-          case Some(Move.Pass)                               => true
-          case None | Some(Move.Play(_)) | Some(Move.Resign) => false
-        }
-    }
-
-  def legalMoves: List[Move] =
-    if (this.isOver) ListBuffer().toList
-    else {
-      val moves = ListBuffer[Move](Move.Pass, Move.Resign)
-      for {
-        row <- 1 to board.row
-        col <- 1 to board.col
-      } {
-        val move = Move.Play(Point(row, col))
-        if (this.isValidMove(move))
```

Comment:
where do legalMoves and winner go? please leave them in. (unless you don't like it and will close the PR otherwise). There's more utility functionality coming up that needs them.

Reply:
It was dead code so I removed them but if they'll be used, I'll leave them. Thanks